HOW_TO_RUN_SAMPLES_WITH_SPRING_BOOT_3.md

```diff
@@ -0,0 +1,28 @@
+# How to run samples with Spring Boot 3?
+
+The current examples are built based on Spring Cloud Azure 4.x by default. If you want to use Spring Cloud Azure 6.x based on Spring Boot 3, this document will tell you how to do it.
```

Comment:
`The current examples are built based on` -> `Samples in this repository is using `